### PR TITLE
Build long prefix package using conda-build 2.x

### PR DESCRIPTION
**Pull request by jjhelmus**

### CODE REVIEW

File: recipe/bld.bat [hunk 1]
```diff
@@ -1,2 +1,2 @@
-if "%ARCH%"=="32" (set CPU_ARCH=i686) else (set CPU_ARCH=x86_64)
-%PKG_NAME%-%PKG_VERSION%-py%PY_VER%-%CPU_ARCH%.exe /userlevel=1 /S /FORCE /D=%PREFIX%
+if "%ARCH%"=="32" (set CPU_ARCH=i686) else (set CPU_ARCH=x86_64)
+%PKG_NAME%-%PKG_VERSION%-py%PY_VER%-%CPU_ARCH%.exe /userlevel=1 /S /FORCE /D=%PREFIX%
```
Comment (jakirkham) [inline recipe/bld.bat]: Fixed the line endings to use CRLF. Hope that is ok.

**Comment by jakirkham** [comment]
@jschueller , could you please take a look at this? It is needed to upgrade `conda-build`.

**Comment by jschueller** [comment]
there's a problem on osx when running a test that involves matplolib and ist macosx backend, maybe related to this:
https://github.com/conda-forge/matplotlib-feedstock/pull/87

**Comment by jakirkham** [comment]
Weird. That problem didn't show up in the [previous build]( https://travis-ci.org/conda-forge/openturns-feedstock/builds/183403638 ).

**Comment by jakirkham** [comment]
Opened issue ( https://github.com/conda-forge/matplotlib-feedstock/issues/94 ) at the `matplotlib` feedstock to track this down. Have cc'd both of you on it.

**Comment by jakirkham** [comment]
This may already be fixed by a recent build of `matplotlib`. Force pushed to see if it makes a difference.

**Comment by jakirkham** [comment]
Looks like everything is now in good shape. What do you think @jschueller ?

**Pull request merged**